Provider/src/FirebirdSql.EntityFrameworkCore.Firebird/Extensions/FbStringExtension.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*
+ *    The contents of this file are subject to the Initial
+ *    Developer's Public License Version 1.0 (the "License");
+ *    you may not use this file except in compliance with the
+ *    License. You may obtain a copy of the License at
+ *    https://github.com/FirebirdSQL/NETProvider/blob/master/license.txt.
+ *
+ *    Software distributed under the License is distributed on
+ *    an "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either
+ *    express or implied. See the License for the specific
+ *    language governing rights and limitations under the License.
+ *
+ *    All Rights Reserved.
+ */
+
+//$Authors = Jiri Cincura (jiri@cincura.net), Rafael Almeida (ralms@ralms.net)
+
+namespace FirebirdSql.EntityFrameworkCore.Firebird.Extensions
+{
+    public static class FbStringExtension
+    {
+	    public static string MaxLength(this string src, int maxLength)
```

Comment:
This can be written with `Substring` directly. No need for extra method.

Reply:
I know this, the question is to keep making several substring.

Reply:
Why? You can call `Substring(Math.Min)` and that's it.

Reply:
I'll be using math in every way.
More if you like it that way. I'll do.
I want those comments for me to improve.

So you think this is better?
identifier.Substring (0, Math.Min (identifier.Length, _options.Settings.ObjectLengthName))

Reply:
Could be. But I think because of [this](https://github.com/cincuranet/FirebirdSql.Data.FirebirdClient/pull/72#discussion_r138803619) the discussion is pointless.